source/rafcon/gui/utils/splash_screen.py

```diff
@@ -45,41 +46,55 @@ def __init__(self, width=530, height=350, contains_image=False):
         if contains_image:
             main_vbox.pack_start(self.image, True, True, 0)
 
-        # add label to display text, the text can be changed by the text() method.
-        # Align it in the middle of the gtk window
-        self.label = Gtk.Label(label="")
-        self.label.set_xalign(0.5)
-        self.label.set_yalign(0.5)
-        main_vbox.pack_start(self.label, False, True, 10)
-        main_vbox.set_spacing(0)
+        if global_design_config.get_config_value("SPLASH_SCREEN_SHOW_TEXT", True):
+            # add label to display text, the text can be changed by the text() method.
+            # Align it in the middle of the gtk window
+            self.label = Gtk.Label(label="")
+            self.label.set_xalign(0.5)
+            self.label.set_yalign(0.5)
+            main_vbox.pack_start(self.label, False, True, 10)
+            main_vbox.set_spacing(0)
+            label_height = global_design_config.get_config_value("SPLASH_SCREEN_LABEL_HEIGHT", 40)
+            self.label.set_size_request(-1, label_height)
 
         if not os.getenv("RAFCON_START_MINIMIZED", False):
             self.show_all()
 
     def set_text(self, text):
+        if not global_design_config.get_config_value("SPLASH_SCREEN_SHOW_TEXT", True):
+            return
         logger.info(text)
         self.label.set_text(text)
-        # include this to give more time to watch
-        # import time
-        # time.sleep(1)
         while Gtk.events_pending():
             Gtk.main_iteration_do(False)
         return
 
     def load_image(self, image_path):
         if image_path:
-            pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_size(image_path, self.get_size()[0] - 50, self.get_size()[1] - 50)
+            horizontal_spacing = global_design_config.get_config_value("SPLASH_SCREEN_HORIZONTAL_SPACING", 50)
+            pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_size(image_path, self.get_size()[0] - horizontal_spacing,
+                                                            self.get_size()[1] - horizontal_spacing)
             self.image.set_from_pixbuf(pixbuf)
             while Gtk.events_pending():
                 Gtk.main_iteration_do(False)
         else:
             logger.debug("Splash screen image path is None")
 
-    def rotate_image(self, random_=True):
-        images = []
-        for image_filename in resource_listdir(rafcon.gui.__name__, os.path.join("assets", "splashscreens")):
-            images.append(resource_filename(rafcon.gui.__name__, os.path.join("assets", "splashscreens", image_filename)))
+    def get_images(self):
+        images = list()
+        if is_custom_design_enabled():
```

Comment:
I think this differentiation is no longer needed, as now also the default design config defines `SPLASH_SCREEN_FOLDER`